src/odemis/driver/tlaptmf.py

```diff
@@ -805,9 +810,9 @@ def _processMessage(self, msg):
                 self._end_motion = time.time() + 1 # 1s move
                 # no output
             else:
-                logging.warning("Message '%X' unknown", mid)
+                logging.warning("Message '%X' unknown", to_str_escape(mid))
         except Exception:
-            logging.exception("Simulator failed on message %X", mid)
+            logging.exception("Simulator failed on message %X", to_str_escape(mid))
```

Comment:
These 2 last changes are wrong. `%X` expects an int, which `mid` is. So no string escaping needed.